src/reference/changes.rst

```diff
@@ -201,9 +201,27 @@ For example, you can (and should) now write:
 
 .. code-block:: diff
 
-   - cylc workflow-state my-workflow --point 20240101 --task mytask --trigger file_created
+   # On the command line
+   - cylc workflow-state my-workflow --point 20240101 --task mytask --message "file 400 has been created"
    + cylc workflow-state my-workflow//20240101/mytask:file_created
```

Comment:
We haven't explained the difference between the message/trigger here, or given them enough information to know that both relate to the same output.

Suggest sticking with `succeeded` for this example (because we know that is the most common use case), then explaining the message -> trigger change after.